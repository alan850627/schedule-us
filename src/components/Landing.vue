<template>
<b-container class="landing">
    <b-row align-h="center">
      <b-col cols="8">
        <b-card>
          <h1>Schedule.us</h1>
          <p>An easy way to find out when everyone is free for your next meeting or event.</p>
          <h4>How it works:</h4>
          <ol>
            <li>Create a new event by picking a range of dates your event could fall on.</li>
            <li>We will generate a link to an event scheduling dashboard for your event.</li>
            <li>Share the scheduling dashboard with your friends to fill out.</li>
            <li>Within the dashboard, you can view everyone's responses and chat.</li>
          </ol>
       </b-card>
      </b-col>
    </b-row>
  </b-container>
</template>

<script>
export default {
  name: 'Landing',

  components: {
  },

  props: {
  },

  computed: {
  },

  data () {
    return {
    }
  },

  methods: {
  },

  mounted () {
  }
}
</script>

<!-- Add "scoped" attribute to limit CSS to this component only -->
<style scoped>

</style>
