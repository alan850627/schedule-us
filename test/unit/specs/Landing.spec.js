import Landing from '@/components/Landing'
import { shallow } from 'vue-test-utils'

describe('Landing.vue', () => {
  it('has the expected html structure', () => {
    let cmp = shallow(Landing, {
    })
    expect(cmp.element).toMatchSnapshot()
    expect(cmp.vm).toBeDefined()
  })
})
