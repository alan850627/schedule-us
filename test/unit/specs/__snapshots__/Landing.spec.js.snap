// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing.vue has the expected html structure 1`] = `
<b-container
  class="landing"
>
  <b-row
    align-h="center"
  >
    <b-col
      cols="8"
    >
      <b-card>
        <h1>
          Schedule.us
        </h1>
         
        <p>
          An easy way to schedule an event with your friends!
        </p>
         
        <h4>
          How to use Schedule.us
        </h4>
         
        <ol>
          <li>
            Login in the top right corner
          </li>
           
          <li>
            Create a new event by choosing a range of dates on which your event could take place.
          </li>
           
          <li>
            A link to the event scheduling dashboard will be generated.
          </li>
           
          <li>
            Share the link with your friends.
          </li>
           
          <li>
            Click on the link to go to the scheduling dashboard, where you can:
          </li>
           
          <ul>
            <li>
              Tell your friends when you are available.
            </li>
             
            <li>
              See when all your friends are available
            </li>
             
            <li>
              Discuss when to schedule the event and email everyone.
            </li>
          </ul>
        </ol>
      </b-card>
    </b-col>
  </b-row>
</b-container>
`;
